### DISCUSSION TITLE
Обновления не прилетают

### DISCUSSION THREAD

**Discussion by anarchyst47** [lAnubisl/LostFilmTorrentsFeed#26] [General]
Собственно, перестало обновляться. И еще был вопрос в отдельной дискуссии про повторное скачивание, тоже не ответили)

**Comment by lAnubisl** [comment]
Можно по-подробнее. Что значит не прилетают? Вроде всё обновляется.

**Comment by anarchyst47** [comment]
Да, как бы вот:

![231](https://user-images.githubusercontent.com/86853763/147413973-5f632e4f-9823-4d94-86db-620b785a8d83.jpg)
![232](https://user-images.githubusercontent.com/86853763/147413979-be50a4b7-df02-4abb-8c6b-e4ccdac6143a.jpg)

**Reply by lAnubisl** [reply]
23.12.2021 Вам должна была прилететь 2я серия сериала 1883. 3я серия ещё не доступна. В чём проблема?

**Reply by anarchyst47** [reply]
> 23.12.2021 Вам должна была прилететь 2я серия сериала 1883. 3я серия ещё не доступна. В чём проблема?
 
дело не в конкретном сериале! на скриншотах видно, что последнее обновление 23 декабря и что я подписан на все) сегодня 26 и я временно вернулся на оригинальный лост, как-то так..

**Reply by lAnubisl** [reply]
Теперь дошло. сейчас проверю в чём дело

**Reply by lAnubisl** [reply]
Починил. Вот что изменилось если кому-то интересно: https://github.com/lAnubisl/LostFilmTorrentsFeed/commit/394a006a495839d6fc77e131821fd88814186f21

**Reply by anarchyst47** [reply]
> Починил. Вот что изменилось если кому-то интересно: [394a006](https://github.com/lAnubisl/LostFilmTorrentsFeed/commit/394a006a495839d6fc77e131821fd88814186f21)

странно, обе эти ленты добавлены у меня в qBittorrent и отрабатывают идентично! может стоит оставить обе, или это уже будет функционал продвинутого клиента?

**Reply by lAnubisl** [reply]
Первый перестал работать

**Comment by anarchyst47** [comment]
Не прилетают:

[Осатанелые (Diabolical)](https://www.lostfilm.tv/series/Diabolical/)
[Загрузка (Upload)](https://www.lostfilm.tv/series/Upload/)
[Последнее Королевство (The Last Kingdom)](https://www.lostfilm.tv/series/The_Last_Kingdom)

**Reply by lAnubisl** [reply]
> Не прилетают:
> 
> [Осатанелые (Diabolical)](https://www.lostfilm.tv/series/Diabolical/) [Загрузка (Upload)](https://www.lostfilm.tv/series/Upload/) [Последнее Королевство (The Last Kingdom)](https://www.lostfilm.tv/series/The_Last_Kingdom)

Скорее всего проблема была связана я тем что сайт не работал, поскольку хостился у меня в шкафу а я был в разъездах. Сейчас работаю над переносом сайта на другой хостинг. Проблема должна была уже решиться но если не решилась - скажите плиз

**Reply by anarchyst47** [reply]
Приветствую, есть ощущение что проверка обновлений происходит раз в час или даже два, хотелось бы минут пять)
К примеру, сейчас (01:12) сервис до сих пор не видит ни **Озарк 4.8** ни **Хромые кони 1.6** а они появились еще вчера.

**Reply by lAnubisl** [reply]
Сейчас проверка происходит раз в 4 часа. Давайте так, Давайте я пока поставлю на 30 минут и посмотрю как пойдёт. Потом наверное придумаю как максимально оптимально сделать проверку раз в 5 минут.

**Reply by anarchyst47** [reply]
раз в 4 часа это вообще несерьезно) что конкретно препятствует более частой проверке, нагрузка на сервер? В качестве хостинга предложил бы рассмотреть Oracle Cloud Free Tier он совсем бесплатный, только для россиян заблокирован, может и для белорусов не уверен. Там на ARM дается достаточно производительный сервер. К сожалению, даже раз в 30 минут лично мне много и придется возвращаться к оригинальным лостфильмовским RSS-кам)

**Reply by lAnubisl** [reply]
Поставил раз в 5 минут. Посмотрю как пойдёт. Я переписал всё на Serverless. сервера нет. Оплата по мере использования ресурсов

**Comment by anarchyst47** [comment]
Приветствую, Амбрелла не прилетает! Выбираю качество, сохраняю изменения и ничего)

**Reply by lAnubisl** [reply]
Починил. Проверьте плиз.

**Comment by anarchyst47** [comment]
Еще момент, из пяти спецэпизодов Экспансии прилетел только пятый, с первого по четвертый проигнорированы!

**Reply by lAnubisl** [reply]
на сайте они есть все. В общий RSS (из которого мы берём информацию) попал только 5й эпизод

**Reply by lAnubisl** [reply]
На сайте, кстати, в списке обновлений есть только эпизоды с 1 по 4.

**Reply by anarchyst47** [reply]
> На сайте, кстати, в списке обновлений есть только эпизоды с 1 по 4.

Пятый есть на сайте, просто он пораньше вышел остальных. А по RSS с какого берете сейчас? И http://insearch.site/rssdd.xml и http://n.retre.org/rssdd.xml содержат все пять эпизодов.

**Reply by lAnubisl** [reply]
Беру из http://insearch.site/rssdd.xml
Скорее всего они не были обработаны так как программа откинула первые 4 эпизода после того как поняла что последний обработанный эпизод 5. Это так работает логика игнирирования дубликатов в летне. Я подумаю как сделать лучше.

**Reply by anarchyst47** [reply]
> Это так работает логика игнирирования дубликатов в летне. Я подумаю как сделать лучше.

Да, помнится я сам просил вас побороть дубли) И по идее, эта логика скоро опять даст сбой, когда они после четвертого сезона "Мир Дикого Запада" вернутся ко второму и третьему. Или она только внутри сезона действует?

**Comment by anarchyst47** [comment]
По всей видимости, каждый новый сериал на ресурсе требует вашего внимания! На этот раз "Человек против пчелы" не сохраняется в изменениях)

**Reply by lAnubisl** [reply]
Обязательно посмотрю что случилось. Завтра. Сегодня доступа к компу нету.

**Reply by lAnubisl** [reply]
Починил

**Comment by anarchyst47** [comment]
Я есть Грут - вышли отдельно пять серий и весь сезон, но прилетела только первая. Не критично, просто информирую)

**Reply by lAnubisl** [reply]
> Я есть Грут - вышли отдельно пять серий и весь сезон, но прилетела только первая. Не критично, просто информирую)

Эпизоды вышли в обратном порядке и в очень кородкое время.
![image](https://user-images.githubusercontent.com/1249671/184105749-73dd0291-ca66-44c0-b7ac-13e03c29b122.png)

Мой сервис не ожидал такой подставы, но сейчас я сохраняю все торрент файлы, так что в будущем можно сделать возможность выкачать предыдущие эпизоды тоже.

**Comment by anarchyst47** [comment]
Полный метр пока решили отсекать или оно само ввиду отсутствия SE?

**Reply by lAnubisl** [reply]
Не совсем понял про 'Полный метр' но без SE ничто не пройдёт фильтр. Да.

**Reply by anarchyst47** [reply]
Я про фильмы:
Добыча
Серый человек

Хотя, "Мир Юрского периода: Господство" присутствует на сервисе без SE, или он вручную добавлен)
По всей видимости, будут другие переводные фильмы! Как потребитель всего контента, приветствовал бы их автоматическое добавление на сервис)

**Comment by anarchyst47** [comment]
Помимо фильмов, игнорится еще один сериал:
https://www.lostfilm.tv/series/Are_You_Afraid_of_the_Dark/

**Reply by lAnubisl** [reply]
Проблема была в вопросительном знаке. Оказалось что в качестве части ключа таблицы его использовать нельзя. Я поправил но из базового RSS этот жпизод уже пропал. Ждём следующего обновления чтобы проверить

**Reply by anarchyst47** [reply]
пока не сохраняет)

**Comment by anarchyst47** [comment]
С Новым вас годом, и спасибо за сервис!
